chipsec/cfg/common.xml

```diff
@@ -170,6 +170,10 @@
     <register name="PCI0.0.0_CAPID0_A"   type="pcicfg" bus="0" dev="0" fun="0" offset="0xE4" size="4" desc="Capabilities A"/>
     <register name="PCI0.0.0_CAPID0_B"   type="pcicfg" bus="0" dev="0" fun="0" offset="0xE8" size="4" desc="Capabilities B"/>
 
+    <!-- IGD registers -->
+    <register name="IGD_DID2" type="pcicfg" bus="0" dev="2" fun="0" offset="2" size="2" desc="Device Identification Number">

```

Comment:
Can you please rename IGD DID register name to "PCI0.2.0_DID" to be consistent with other devices